ui/src/components/inputs/InputsForm.vue

```diff
@@ -21,8 +37,7 @@
                 :full-height="false"
                 :input="true"
                 :navbar="false"
-                v-if="input.type === 'ENUM' || input.type === 'SELECT'"
-                :data-test-id="`input-form-${input.id}`"
+                v-else-if="input.type === 'ENUM' || input.type === 'SELECT'"
```

Comment:
- Put the radio group below the el-select
- Add the radio condition in both v-if of the el-select and the el-radio-group